src/dso_api/dynamic_api/filterset.py

```diff
@@ -51,15 +51,68 @@ def filterset_factory(model: Type[DynamicModel]) -> Type[DynamicFilterSet]:
     # Determine which fields are included:
     # Excluding geometry fields for now, as the default filter only performs exact matches.
     # This isn't useful for polygon fields, and excluding it avoids support issues later.
-    fields = {f.attname: _get_field_lookups(f) for f in model._meta.get_fields()}
+    fields = {
+        f.attname: _get_field_lookups(f) for f in model._meta.get_fields()
+        if isinstance(f, (models.fields.Field, models.fields.related.ForeignKey))
```

Comment:
Even nieuwsgierig: wanneer is iets in `_meta.get_fields()` geen `Field`?
Of ben je hier `f.is_relation` aan het excluden?

Reply:
Klopt, doel hier om alle `backwards` relaties weg te gooien (voor deze PR waren ze niet gedefineerd) en later opnieuw opbouwen.

Anders filterset failt met `Incorrect fields provided` fout.